tools/codegen/config.yml

```diff
@@ -543,3 +543,53 @@ resources:
           sensitive: true
         roles:
           type: set
+
+  alert_configuration_api:
+    read:
+      path: /api/atlas/v2/groups/{groupId}/alertConfigs/{alertConfigId}
+      method: GET
+    create:
+      path: /api/atlas/v2/groups/{groupId}/alertConfigs
+      method: POST
+    update:
+      path: /api/atlas/v2/groups/{groupId}/alertConfigs/{alertConfigId}
+      method: PUT
+    delete:
+      path: /api/atlas/v2/groups/{groupId}/alertConfigs/{alertConfigId}
+      method: DELETE
+    schema:
+      aliases:
+        alertConfigId: id # path param name does not match the API response property
+      overrides:
+        notifications.notifier_id:
+          computability:
+            optional: true
+            computed: true
+        notifications.delay_min:
+          computability:
+            optional: true
+            computed: true
+        notifications.interval_min:
+          computability:
+            optional: true
+            computed: true
```

Comment:
so these are both getting default values returned from the API right?

Reply:
Given API behaviour should not be common scenario: Potentially a user that defined the attribute and then removes it, leaving the config with no attribute but the state will still have it present.